src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import Display from './display';
+import ButtonPanel from './buttonPanel';
+
+function App() {
+  return (
+    <div className="container d-flex flex-column app rounded">
+      <Display value="0" />
+      <ButtonPanel />
+    </div>
```

Comment:
## Implementation of the rendered component's children with React Fragments.
- [ ]  As required of you from this ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)), these children of this component ought to have been wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) `short syntax` instead.